examples/miscellaneous/miscellaneous_ex2/miscellaneous_ex2.C

```diff
@@ -543,9 +548,9 @@ void add_M_C_K_helmholtz(EquationSystems & es,
 
   // Compute the scale values for the different operators.
   const Number scale_stiffness (1., 0.);
-  const Number scale_damping (0., omega);
-  const Number scale_mass (-k*k, 0.);
-  const Number scale_rhs (0., -(rho*omega));
+  const Number scale_damping=sqrt(-(Number)1.)*omega;
```

Comment:
Why is this preferable to the two-argument constructor?  It'll all compile to the same thing in the end, but I find the former easier to read.

Reply:
yes, it would be; but since omega is complex as well, the two-argument constructor doesn't work. 
But I'll add a variable for i; than it looks already better.